.github/workflows/docker_publish.yml

```
@@ -148,46 +174,121 @@ jobs:
+          tags: ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}-${{ matrix.compiler}}-ext-hdf5_${{ matrix.hdf5_versions}}-moab_${{ matrix.moab_versions }}
 
+  housekeeping:
+    needs: [build-housekeeping-img]
+    runs-on: ubuntu-latest
+    env:
+      hdf5_versions: 1.10.4
+      hdf5_build_dir: hdf5_build_dir    
+    strategy:
+      matrix:
+        ubuntu_versions : [18.04,]
+    container: ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}-housekeeping:latest
 
-  build_and_push_moab:
+    name: Running Housekeeping scrips
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Setup
+        run: |
+          mkdir /root/build_dir/
+          echo "REPO_SLUG=${GITHUB_REPOSITORY}" >> $GITHUB_ENV
+          echo "PULL_REQUEST=$(echo  $GITHUB_REF | cut -d"/" -f3)" >> $GITHUB_ENV
+          ln -s $GITHUB_WORKSPACE /root/build_dir/DAGMC
+
+      - name: Run Housekeeping
+        run: |
+          cd $GITHUB_WORKSPACE
+          /bin/bash "CI/scripts/housekeeping.sh"
+      
+  test_img:
+    # if: ${{ contains(github.event_name, 'pull_request') }}
+    needs: [build-moab-img]
     runs-on: ubuntu-latest
-    needs: build_and_push_hdf5
+    env:
+      hdf5_versions: 1.10.4
+      hdf5_build_dir: hdf5_build_dir
+
     strategy:
       matrix:
         ubuntu_versions : [
-          16.04,
           18.04,
-        ]
+          20.04,
+          ]
         compiler : [
           gcc,
           clang,
-        ]
+          ]
         hdf5_versions : [
-          1.10.4,
+         1.10.4,
         ]
         moab_versions : [
-          9c96d17,
-          develop,
-          master,
+         9c96d17,
+         develop,
+         master,
         ]
+    container: ghcr.io/${{ github.repository_owner }}/dagmc-ci-ubuntu-${{ matrix.ubuntu_versions }}-${{ matrix.compiler}}-ext-hdf5_${{ matrix.hdf5_versions}}-moab_${{ matrix.moab_versions }}:latest
 
+    name: Installing and Testing DAGMC
     steps:
-
       - name: Checkout repository
         uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Setup
+        run: |
+          echo "MOAB_VERSION=${{ matrix.moab_versions }}" >> $GITHUB_ENV
+          echo "COMPILER=${{ matrix.compiler }}" >> $GITHUB_ENV
+          echo "HDF5_VERSION=${{ matrix.hdf5_versions }}" >> $GITHUB_ENV
+          echo "REPO_SLUG=${GITHUB_REPOSITORY}" >> $GITHUB_ENV
+          echo "PULL_REQUEST=$(echo  $GITHUB_REF | cut -d"/" -f3)" >> $GITHUB_ENV
+          echo "DOUBLE_DOWN="OFF"" >> $GITHUB_ENV
+          ln -s $GITHUB_WORKSPACE /root/build_dir/DAGMC
+
+      - name: Install DAGMC
+        run: |
+          cd $GITHUB_WORKSPACE
+          /bin/bash "CI/scripts/install.sh"
+
+      - name: Run DAGMC Test
+        run: |
+          cd $GITHUB_WORKSPACE
+          /bin/bash "/scripts/tests.sh"
```

This is the wrong path.... maybe
```suggestion
          /bin/bash "CI/scripts/tests.sh"
```